prowler/lib/check/check.py

```diff
@@ -593,12 +593,17 @@ def execute_checks(
                     service,
                     check_name,
                     global_provider,
-                    services_executed,
-                    checks_executed,
                     custom_checks_metadata,
                 )
                 all_findings.extend(check_findings)
 
+                # Update Audit Status
```

Comment:
Why are you moving this outside of the `execute` function? Is there any reason impacting the scan class?

Reply:
Nop, it's just to modularize the code